### ISSUE TITLE
docker-compose.yml on Ubuntu 20.04 

### ISSUE THREAD

**Issue by theitmachine** [getumbrel/umbrel#472]
Strange error on a brand new 20.04.2
```

No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal

```

docker run hello-world works fine.

```

sudo ./scripts/start

======================================
============= STARTING ===============
============== UMBREL ================
======================================

Setting environment variables...

Starting karen...

Starting backup monitor...

Starting decoy backup trigger...


Starting Docker services...

ERROR: 
        Can't find a suitable configuration file in this directory or any
        parent. Are you in the right directory?

        Supported filenames: docker-compose.yml, docker-compose.yaml
        

docker-compose version
docker-compose version 1.25.5, build unknown
docker-py version: 4.2.1
CPython version: 3.5.2
OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016

docker --version
Docker version 19.03.11, build dd360c7


```
Renaming, changing permissions, doesn't seem to matter.

**Comment by maltokyo** [comment]
are you sure you didnt remove somehow the docker-compose.yml file?  Can you pwd and see what directory you are in, and check that docker-compose.yml is in exactly the same directory?

**Comment by theitmachine** [comment]
Yup, didn't change anything in the directory tree, docker-compose is under $UMBREL_ROOT which I have set to /opt/umbrel 

For testing, I added echo lines to the start script to make sure the variables being set made sense. They do

I've removed the working directory and re-downloaded a few times now per:
`curl -L https://github.com/getumbrel/umbrel/archive/v0.3.2.tar.gz | tar -xz --strip-components=1`

```
/opt/umbrel$ ls
LICENSE.md  SECURITY.md  apps  bitcoin  docker-compose.yml  events     karen  logs   scripts   templates
README.md   app-data     bin   db       electrs             info.json  lnd    nginx  statuses  tor
```

```
/opt/umbrel$ sudo ./scripts/start 

======================================
============= STARTING ===============
============== UMBREL ================
======================================

Setting environment variables...

Starting karen...

Starting backup monitor...

Starting decoy backup trigger...


Starting Docker services...

ERROR: 
        Can't find a suitable configuration file in this directory or any
        parent. Are you in the right directory?

        Supported filenames: docker-compose.yml, docker-compose.yaml

```

**Comment by louneskmt** [comment]
Can you check the output of this?
```
cd /opt/umbrel && docker-compose ps
```

**Comment by theitmachine** [comment]
Curious, that took a good 30 seconds to run and the output was the same. 

```
cd /opt/umbrel && docker-compose ps


ERROR: 
        Can't find a suitable configuration file in this directory or any
        parent. Are you in the right directory?

        Supported filenames: docker-compose.yml, docker-compose.yaml
        
```


On a whim i'm going to run a few other projects that use docker-compose later today and see if this isn't some sort of dependency issue

**Comment by theitmachine** [comment]
Fixed it.
The problem is with the snap install docker setup Ubuntu 20.04 can do on install 
During install  you are presented with a menu to add additional services. Docker was on that list and both docker-compose and docker responded to version calls as expected. But, docker-compose wasn't able to run on any project I tried.

I removed both.

`sudo snap remove docker`

Then installed docker and docker-compose per the repository methods I usually do.

Links to that install process for anyone else reading:
[https://docs.docker.com/engine/install/ubuntu/](url)
[https://docs.docker.com/search/?q=install%20docker-compose](url)

After all that, docker-compose ran fine. Umbrel is syncing now.
